assets/blocks/reader-registration/index.php

```diff
@@ -113,8 +113,11 @@ function render_block( $attrs, $content ) {
 
 	// phpcs:disable WordPress.Security.NonceVerification.Recommended
 	if (
-		\is_user_logged_in() ||
-		( isset( $_GET['newspack_reader'] ) && absint( $_GET['newspack_reader'] ) )
+		! \is_preview() &&
```

Comment:
This will also allow the block to be previewed via the Campaigns wizard (e.g. previewing as a segment):

```suggestion
		! \is_preview() &&
		( ! method_exists( '\Newspack_Popups', 'is_preview_request' ) || ! \Newspack_Popups::is_preview_request() ) &&
```